Lib/test/test_ipaddress.py

```diff
@@ -2277,6 +2277,39 @@ def testReservedIpv4(self):
         self.assertEqual(False, ipaddress.ip_address('128.0.0.0').is_loopback)
         self.assertEqual(True, ipaddress.ip_network('0.0.0.0').is_unspecified)
 
+    def testPrivateNetworks(self):
+        self.assertEqual(False, ipaddress.ip_network("0.0.0.0/0").is_private)
```

Comment:
A `/0` network is a weird concept.  0 bits so it's the equivalent of a null non-address or a broadcast only address?  that makes me think of it more as `NaN` where it can't be private or public and the very question is irrelevant.  regardless, i'm fine with this returning False on is_private.  That is certainly more accurate than True.

Reply:
for others: of all of these assertions, this is the only one that changes behavior with this PR.